frontend/src/component/UIFactory.tsx

```diff
@@ -50,6 +51,20 @@ export class UIFactory {
     );
   }
 
+  public createViewChangeLogButton(props: any): ReactElement {
+    if (!this.featureDecisionService.includeViewChangeLogButton()) {
+      return <div />;
+    }
+    return (
+      <Fragment>
+        <div className={'row view-changelog'} onClick={props.openModal}>
+          <a href={'/#'}>View Changelog</a>
+        </div>
+        <ChangeLogModal {...props} />
+      </Fragment>
```

Comment:
Let's extract this into its own file.

Reply:
"extract this into its own file" , you mean return this:
```
      <Fragment>
        <ViewChangeLogButton onClick={props.openModal} />
        <ChangeLogModal {...props} />
      </Fragment>
```

or this:
```
        <ViewChangeLogButton onClick={props.openModal} {...props} />

```

Reply:
@kapillamba4 The second one.